static-analysis/teammates-macker.xml

```diff
@@ -78,21 +78,47 @@
                         <allow>
                             <from>
                                 <include class="${ui}.controller.**" />
-                                <include class="${ui}.newcontroller.**" />
+                                <include class="${ui}.webapi.action.**" />
                                 <include class="${ui}.automated.**" />
                             </from>
                             <to class="${logic}.api.**" />
                         </allow>
                     </deny>
                 </access-rule>
             </ruleset>
+            <ruleset name="UiComponent: API output is only used in Web API package">
+                <access-rule>
+                    <deny>
+                        <from class="${ui}.**" >
```

Comment:
This `from` is redundant?

Reply:
Don't think so: it blocks the `ui.automated` package from accessing.

Reply:
`ui.automated` needs access to API output?

Reply:
No, it does not. Although technically speaking `ui.automated` is also a RESTful API package, we don't require them to have properly defined output at least for now.